02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,69 @@
+.title{
+  float: left;
+  padding-top: 0px;
+  padding-right: 50px;
+  padding-bottom: 0px;
+  padding-left: 0px;
+  box-sizing: border-box;
+}
+
+.title h2{
+  margin: 0px;
+  text-align: left;
+  font-size: 26px;
+}
+
+h2{
+  font-weight: 500;
+}
+
+h2{
+  font-family: 'GT-Sectra';
+  line-height: 150%;
+}
+
+p{
+  width: 50%;
+  float: right;
+  text-align: left;
+}
+p {
+  margin:0 auto 25px;
+  max-width: 800;
+  color: rgba(0,0,95,0.75);
+  font-family: 'GT-Sectra';
+  font-weight: 300;
+}
+#comtainer{
```

Comment:
this code looks professional! great job. I think this id should be 'contain', not 'comtainer'. overall, a fantastic site. i look forward to seeing what you build in the future.